insta-clone/src/components/Login/Login.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+
+
+class Login extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            username: '',
+            password: ''
+        } 
+    }
+
+    componentDidMount(){
+        
+    }
+
+    userChangeHandler = event => {
```

Comment:
Another way to do this is to add a name to the input as I will comment below and then have only 1 onChange handler for all inputs. It would look like this: `onChangeHandler = event => {this.setState({event.target.name: event.target.value})}`